### PR TITLE
Add adaptive authentication functional library

### PR DESCRIPTION
**Pull request by senthalan**

### CODE REVIEW

File: en/mkdocs.yml [hunk 1]
```diff
@@ -108,6 +108,7 @@ nav:
           - 'Configuring ACR-Based Adaptive Authentication': learn/configuring-acr-based-adaptive-authentication.md
           - 'Configuring Risk-Based Adaptive Authentication': learn/configuring-risk-based-adaptive-authentication.md
           - 'Using WSO2 Stream Processor for Adaptive Authentication': learn/using-wso2-stream-processor-for-adaptive-authentication.md
+        - 'Function Library for Adaptive Authentication': tutorials/function-library-for-adaptive-authentication.md
```
Comment (gomathyk) [inline en/mkdocs.yml]: Please replace tutorials with learn

File: en/docs/tutorials/function-library-for-adaptive-authentication.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+        }
+        return age;
+    }
+     module.exports.getAge = getAge;
+    ```
+
+2. To try out adaptive authentication with WSO2 Identity Server, you have to set up 
+a sample application to test adaptive authentication scenarios, and configure a service provider 
+for adaptive authentication in WSO2 Identity Server. 
+For instructions on how to do this, see [Configuring a Service Provider for Adaptive Authentication](../../tutorials/configuring-a-service-provider-for-adaptive-authentication).
```
Comment (gomathyk) [inline en/docs/tutorials/function-library-for-adaptive-authentication.md]: Please replace tutorials with learn.

File: en/docs/tutorials/function-library-for-adaptive-authentication.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+
+
+### Adding a function library
+
+Follow the steps below to add a function library using the Management Console,
+
+1. Sign in to in to the Management Console.
+2. On the Main menu, click Manage > Function Libraries >  Add.
+   
+    ![](/assets/img/tutorials/functional-lib-menu-manage.png)
```
Comment (gomathyk) [inline en/docs/tutorials/function-library-for-adaptive-authentication.md]: please correct the formats of all the image paths to (../assets/img/*/*.png)

File: en/docs/tutorials/function-library-for-adaptive-authentication.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
+### Introduction
```
Comment (YvonneW) [inline en/docs/tutorials/function-library-for-adaptive-authentication.md]: Let's rename this to # Adaptive Authentication with Function Library

**Pull request merged**